components/Typography/src/heading.scss

```diff
@@ -47,6 +47,14 @@
   --utrecht-heading-5-margin-block-start: var(--denhaag-space-block-xl);
 }
 
+.utrecht-heading-1 + *:not(.denhaag-button),
+.utrecht-heading-2 + *:not(.denhaag-button),
+.utrecht-heading-3 + *:not(.denhaag-button),
```

Comment:
This could be a lot shorter:
`[class^='utrecht-heading'] + *:not(.denhaag-button), [class*=' utrecht-heading'] + *:not(.denhaag-button)`